lib/apiservers/engine/backends/container.go

```diff
@@ -824,6 +828,21 @@ func (c *Container) imageExist(imageID string) (storeName string, err error) {
 	return host, nil
 }
 
+func createNewAttachClientWithTimeouts(connectTimeout, responseTimeout, responseHeaderTimeout time.Duration) (*client.PortLayer, *httpclient.Transport) {
```

Comment:
Yeah, good catch.  We're not interested in that one right now, but since I declare in params, I should use it.